bindings/node.js/Makefile

```diff
@@ -44,17 +44,6 @@ build: install clean
 	@$(YARN) node-gyp --loglevel=warn configure
 	@$(YARN) node-gyp --loglevel=warn build
 
-# Bundle the distribution, also helpful for cross compatibility checks
-.PHONY: bundle
-bundle: build
-	@mkdir dist
-	@mv deps dist
-	@cp -r lib dist/lib
-	@cp -r src dist/src
-	@cp README.md dist/README.md
-	@cp package.json dist/package.json
-	@cp binding.gyp dist/binding.gyp
```

Comment:
The `files` key in the `package.json` is what controls this behavior.  There is no need to build the `dist` folder.  Went with this before because you guys seemed to like it so rode the wave but the error showed up so switched this to the canonical method of using the `files` key to control what gets bundled in the npm tarball